src/apps/answers/service.py

```diff
@@ -573,7 +573,13 @@ async def get_export_data(
         if not access:
             allowed_respondents = [self.user_id]
         elif access.role == Role.REVIEWER:
-            allowed_respondents = access.reviewer_respondents  # type: ignore[assignment] # noqa: E501
+            if (
+                isinstance(access.reviewer_respondents, list)
+                and len(access.reviewer_respondents) > 0
```

Comment:
You can just do
```
if access.reviewer_respondents :
```

Reply:
I tried, but mypy was giving me an error that property can be always true in boolean context